templates/common/infra/bicep/core/host/container-app.bicep

```diff
@@ -58,6 +90,10 @@ resource app 'Microsoft.App/containerApps@2022-03-01' = {
           }
         }
       ]
+      scale: {
+        minReplicas: containerMaxReplicas > 0 ? containerMaxReplicas : 0
```

Comment:
I think we might have inverted these?  I would have expected that `minReplicas`'s value would be set based on `containerMinReplicas` not "Max" as you have here.  If we use the `@minValue` decorators on the parameters (as I noted above) to enforce these conditions, I think we can get rid of this logic here.

Reply:
fixed. removed the condition here and added minValue constraints to the params.